### PR TITLE
chore(mixins): remove `video` mixin

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Removes a mixin from video component before migration to Vue 3
* Clean up unused properties, refactor migrated code
* Lint files

### 🖼️ Screenshots / Screencasts

No visual changes

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] ⛑️ Tests are included or not possible

### CODE REVIEW

**Comment by DorraJaouad** [comment]
Why not migrating it to Vue 3's Composition API ?

**Comment by Antreesy** [comment]
> Why not migrating it to Vue 3's Composition API ?

Doesn't make sense:
- data: `mouseover` only needed, and it would be a lot of boilerplate code to achive the same result with composable
- props: couldn't be defined in composable +  it's hard to find them, if they are on a different file
- computed: styles for classes are duplicated in components, couldn't be moved to composable, one-line in template
- methods: after refactoring it's a one-line in template, so same as computed

File: src/components/CallView/Grid/Grid.vue [hunk 1]
```diff
@@ -67,8 +67,6 @@
 									:is-stripe="isStripe"
 									:is-promoted="sharedDatas[callParticipantModel.attributes.peerId].promoted"
 									:is-selected="isSelected(callParticipantModel)"
-									:fit-video="false"
```
Comment (DorraJaouad) [inline src/components/CallView/Grid/Grid.vue:70]: As you are not passing it as a prop, remove it from Video.

Comment (Antreesy) [reply src/components/CallView/Grid/Grid.vue:70]: It could passed, I just don't see a reason to keep this line here, as fit Video prop is false by default

Comment (DorraJaouad) [reply src/components/CallView/Grid/Grid.vue:70]: oops, found the other instance of the component. nvm

**Review by DorraJaouad — APPROVED** [review]
It is good to go.

**Pull request merged**